### DISCUSSION TITLE
ORC plant configuration

### DISCUSSION THREAD

**Discussion by malcolm-dsider** [NatLabRockies/GEOPHIRES-X#35] [General]
Another question, regarding the GEOPHIRES end-use model, what sort of ORC plant configuration is used for the subcritical and supercritical cycles? Can I change the end-use model for other ORC configurations?

@kfbeckers - do you have a response to this question?

**Comment by kfbeckers** [comment]
The original ORC models were developed in AspenPlus. They assume standard ORC cycle with heat recuperator and air cooled condensers. Various refrigerants were considered at different operating temperatures. More background information is available in section 4.3.3 in https://ecommons.cornell.edu/items/ca2cdbf7-afe3-4df0-98e6-ee274f7fc5e7 and sections 5.3 and 5.4 in https://dspace.mit.edu/handle/1721.1/51671 . Changing the ORC configuration would require revisiting these power plant models. As part of a recent NREL project, we developed new ORC models in IPSEpro (https://pangea.stanford.edu/ERE/db/GeoConf/papers/SGW/2022/Beckers.pdf).
